### PR TITLE
use notify_failure

### PR DESCRIPTION
**Pull request by braver**

### CODE REVIEW

File: linter.py [hunk 1]
```diff
@@ -24,31 +26,33 @@ def find_errors(self, output):
...
+        deprecations and other feedback about the config.
         """
         data = None
         match = self.crash_regex.match(output)
 
         if match:
             msg = "Stylelint crashed: %s" % match.group(1)
-            yield (match, 0, None, "Error", "", msg, None)
+            self.notify_failure()
+            yield (match, 0, None, "", "Warning", msg, None)
```
Comment (kaste) [inline linter.py]: Do not yield a 'problem'. Call `self.notify_failure(); logger.error(output); return`. `logger.error` for inda face or `logger.warning` just to log it. (So the user has to open the Sublime console.)

**Review by kaste — CHANGES_REQUESTED** [review]
Do not call `return super().find_errors(output)`.

Do not yield `(True, ...)`, but log these warnings or errors.

**Comment by braver** [comment]
Ah, that helps, thanks. PTAL?

File: linter.py [hunk 2]
```diff
@@ -24,32 +26,35 @@ def find_errors(self, output):
...
+            logger.warning(e)
+            self.notify_failure()
 
         if data and 'invalidOptionWarnings' in data:
             for option in data['invalidOptionWarnings']:
                 text = option['text']
-
-                yield (True, 0, None, "Error", "", text, None)
+                logger.warning(text)
+                self.notify_failure()
```
Comment (kaste) [inline linter.py]: If possible do not call `notify_failure` in a loop?

File: linter.py [hunk 3]
```diff
@@ -62,5 +67,3 @@ def find_errors(self, output):
                     yield (True, line, col, "", type, text, None)
                 else:
                     yield (True, line, col, type, "", text, None)
```
Comment (kaste) [inline linter.py]: Instead of True you usually pass an error dict (here: `warning`) down.

**Review by kaste — APPROVED** [review]

**Comment by braver** [comment]
Oh, you already approved. Made some changes with your feedback. Works a charm, thanks.

**Comment by kaste** [comment]
Don't forget you can't release bc it uses API from SL master.

**Comment by braver** [comment]
I know, leaving the PR open until we're ready for it.

**Pull request merged**